data/json/npcs/other/TALK_NPC_PREPPER_SURVIVOR.json

```diff
@@ -0,0 +1,85 @@
+    "responses": [
+      {
+        "text": "&Hold up your hands.  \"Don't worry, I'm not going to hurt you\"",
+        "topic": "TALK_NPC_PREPPER_SURVIVOR_CALM",
+        "condition": { "not": { "npc_has_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" } }
+      },
+      {
+        "text": "Hand over your stuff!  Don't make any sudden moves, or you die!",
+        "trial": { "type": "INTIMIDATE", "difficulty": 30 },
+        "success": { "topic": "TALK_WEAPON_DROPPED", "effect": "drop_weapon", "opinion": { "trust": -4, "fear": 3 } },
+        "failure": { "topic": "TALK_DONE", "effect": "hostile" },
+        "condition": { "not": { "npc_has_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" } }
+      },
+      {
+        "text": "<greet>",
+        "topic": "TALK_NPC_PREPPER_SURVIVOR_INTRO",
+        "condition": { "npc_has_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" }
+      },
+      { "text": "Bye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NPC_PREPPER_SURVIVOR_CALM",
+    "dynamic_line": "Oh, you're not one of the <zombies>.  Come on in, just don't cause any trouble.",
+    "responses": [
+      { "text": "Alright, let's chat.", "topic": "TALK_NPC_PREPPER_SURVIVOR_INTRO" },
+      { "text": "I'll be back later.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NPC_PREPPER_SURVIVOR_INTRO",
+    "dynamic_line": "What can I do for you?",
+    "responses": [
+      { "text": "What are you doing here?", "topic": "TALK_NPC_PREPPER_SURVIVOR_STORY" },
+      {
+        "text": "Would you be interested in bartering?",
+        "topic": "TALK_NPC_PREPPER_SURVIVOR_INTRO",
+        "effect": "start_trade"
+      },
+      { "text": "Can I stay here for a while?", "topic": "TALK_NPC_PREPPER_SURVIVOR_STAY" },
+      {
+        "text": "Why don't you come with me?",
+        "trial": { "type": "PERSUADE", "difficulty": -15, "mod": [ [ "value", 2 ] ] },
+        "success": { "topic": "TALK_AGREE_FOLLOW", "effect": "follow", "opinion": { "trust": 1, "value": 1 } },
+        "failure": { "topic": "TALK_DENY_FOLLOW", "effect": "deny_follow", "opinion": { "trust": 0, "fear": 0 } }
+      },
+      {
+        "text": "Any jobs you need done?",
+        "condition": { "and": [ { "not": "has_assigned_mission" }, { "not": "has_many_assigned_missions" } ] },
+        "topic": "TALK_MISSION_LIST"
+      },
+      { "text": "I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NPC_PREPPER_SURVIVOR_STORY",
+    "dynamic_line": "They all said I was crazy for <swear> preparing for <the_cataclysm>.  Now, they're all dead, and I'm still kickin'.  When the riots turned to hordes, I was ready; all I had to do was dodge a few <zombies> to get back to my bastion.  I've been living safely here ever since.",
```

Comment:
I dont know about `<swear>` here since it may evaluate to "cursed" or "damn" which does not really fit this sentence. Moving it between `<the_cataclysm>` would not work either since it needs to be `<the> <swear> <cataclysm>` which we do not have.

Reply:
I've removed it.